src/Common/src/System/Net/ContextAwareResult.cs

```diff
@@ -153,7 +153,7 @@ internal ExecutionContext ContextCopy
                 ExecutionContext context = _context;
                 if (context != null)
                 {
-                    return context.CreateCopy();
+                    return context; // No need to copy on CoreCLR; ExecutionContext is immutable
```

Comment:
IIRC this code is shared with the desktop implementation of System.Net. @CIPop could also confirm.